README.md

```diff
@@ -43,7 +43,7 @@ React:
 | `enzyme-adapter-react-16` | `^16.4.0-0` |
 | `enzyme-adapter-react-16.3` | `~16.3.0-0` |
 | `enzyme-adapter-react-16.2` | `~16.2` |
-| `enzyme-adapter-react-16.1` | `~16.0.0-0 || ~16.1` |
+| `enzyme-adapter-react-16.1` | `16.0.0-0 - 16.1.x` |
```

Comment:
I see how the pipes could cause a problem, but it’s essential to keep the same semver range here that the packages use. Can we find a way to make the pipes work?